continue-reading.html

```diff
@@ -4,10 +4,46 @@
     <title>Continue Reading</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link href="/style3.css" rel="stylesheet" type="text/css">
   </head>
+  <style>
+    a {color: Black;}
+    a:hover {color: yellow};
+  </style>
   <body>
     <section id="header"></section>
     <section id="main"></section>
     <section id="footer"></section>
+    <div class="container">
+   
+    <div class='item header'>
+    <a href="/index.html"><h1>Continue Reading</h1></a></div>
+    <div class="container3">
+      <div class="item content-1">
+        <h1>Yoda</h1>
+        <p>Yoda was a legendary Jedi Master and stronger than most in his connection with the Force. Small in size but wise and powerful, he trained Jedi for over 800 years, playing integral roles in the Clone Wars, the instruction of Luke Skywalker, and unlocking the path to immortality.</p>
+        
+        <h2>Obi-Wan Kenobi</h2>
+        <p>A legendary Jedi Master, Obi-Wan Kenobi was a noble man and gifted in the ways of the Force. He trained Anakin Skywalker, served as a general in the Republic Army during the Clone Wars, and guided Luke Skywalker as a mentor.</p>
+        
+        <h3>Mace Windu</h3>
+        <p>A grim Jedi Master with an amethyst-bladed lightsaber, Mace Windu was the champion of the Jedi Order, with little tolerance for the failings of the Senate, the arguments of politicians, or the opinions of rebellious Jedi. As the Clone Wars intensified, Mace sensed the dark side of the Force at work, and knew the Jedi's enemies were plotting to destroy the Order and end its stewardship of the galaxy.</p>
+       
+        <h4>Qui-Gon Jinn</h4>
+        <p>A venerable if maverick Jedi Master, Qui-Gon Jinn was a student of the living Force. Qui-Gon lived for the moment, espousing a philosophy of "feel, don't think -- use your instincts." On Tatooine, Qui-Gon discovered a young slave boy named Anakin Skywalker who was strong in the Force. Sensing the boy's potential, Qui-Gon liberated Anakin from slavery. The Jedi Master presented Anakin to the Jedi Council, but they deemed the boy too old to begin training and dangerously full of fear and anger. They refused to allow Qui-Gon to train Anakin, but rescinded their decision to fulfill Qui-Gon's dying wish.</p>

```

Comment:
Creating these sections with `<h1>` through `<h4>` is going to create 4 different sized headers. If you wanted the titles to be uniform, just keep using your `<h1>` tag, (or whichever size you want them to be uniform to)